server/mail_templates/ingestDatasetConfirmation.mako

```diff
@@ -51,6 +51,14 @@ Thank you for contributing a ZIP image batch to the ISIC Archive.
     </tbody>
 </table>
 
+% if failedImages:
+    <h4>Failures</h4>
+    <br>One or more files within your dataset failed to be completely imported into the archive:<br>
+    % for failedImage in failedImages:
+        <a href="${host}/girder#item/${failedImage['_id']}">${failedImage['name']}</a><br>
```

Comment:
Use `failedImage['privateMeta']['originalFilename']` for the name. The `'name'` like `ISIC_1234567` is not meaningful in this context